stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterExecutorHolder.java

```diff
@@ -16,5 +18,21 @@
  * A holder class for the executor service used for pretty printing related tasks
  */
 class AsyncPrettyPrinterExecutorHolder {
```

Comment:
this class should also be `final` and have a private, empty, and parameterless constructor